util/getUniqueRoles.js

```diff
@@ -0,0 +1,14 @@
+export default function getUniqueRoles(teamMembers) {
+  const teamMemberIdArray = Object.keys(teamMembers);
+  const uniqueRoles = [];
+
+  for (let i = 0; i < teamMemberIdArray.length; i += 1) {
+    const thisRole = teamMembers[teamMemberIdArray[i]].jobTitle;
+
+    if (!uniqueRoles.includes(thisRole)) {
+      uniqueRoles.push(thisRole);
+    }
+  }
```

Comment:
We can do some similar stuff with `Set` and mapping to simplify this as well:

```js
const roles = Object.keys(teamMembers).map((key) => teamMembers[key].jobTitle);

const uniqueRoles = [...new Set(roles)];
```

I can't overstate just how magical `Set` is.